consensus/src/chained_bft/consensus_types/block.rs

```diff
@@ -142,8 +171,10 @@ where
             height: 0,
             timestamp_usecs: 0, // The beginning of UNIX TIME
             quorum_cert: genesis_quorum_cert,
-            author: genesis_validator_signer.author(),
-            signature,
+            block_source: BlockSource::Proposal {
```

Comment:
Not for this PR, but in future work, I think it would be nice for genesis blocks to also follow the policy of of NilBlock not having an author & signature.